### ISSUE TITLE
Feature Request: Planning and saving a route with OSMand on iOS

### ISSUE THREAD

**Issue by bowman68** [osmandapp/OsmAnd-ios#590]
Hi,

i can't believe it - it seems to me that there's actually no way to plan *and save* a route in OSMand on iOS. On my Android phone, it's simple and easy: I can plan a route from A to B, insert intermediate waypoints on the map and save the calculated route as a .gpx-track to be followed later. Easy as that. On my iPadPro, it's driving me nuts: The route planning works the same, but there's no way to save it as a .gpx-track. I had the hope that at least it should be possible to define and save a number of waypoints that can be loaded in the navigation screen, but - nope. I've tried to figure out how to use the trip planning function, but if i manage not to lose my mind with all the nested dialogues and edit-functions, the best i can create are staight linked waypoints that don't allow any turn-by-turn navigation.

So basically: Is there any chance to get the route-planning functions of OSMand on Android on the iOS-Version? OSMand is really great on Android, but i can't believe that my only option to get these basic route planning functions on a tablet in my car is to buy a Galaxy Tab. To me, planning *and saving* a route seems to be a quite essential function for a navigation tool, so i wonder why it is missing in the iOS-version of OsmAnd...

**Comment by dmpr0** [comment]
Hello, @bowman68!
We are currently finishing work on adding Export feature to calculated route. You can try it now in TestFlight version https://testflight.apple.com/join/7poGNCKy, or wait until release, we are planning to release the new iOS version in a few weeks.

**Comment by bowman68** [comment]
Hello Dimitry,

thank you so much for the link to the new Beta! It adds exactly was i was searching for - excellent work! I still have to test following saved and imported .gpx-track in the field, but right now i’m really glad that it adds this central feature.  :-)

Best,
Sigmar

**Comment by vshcherb** [comment]
Fixed - Coming next release
